projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -17,30 +17,78 @@
 
 <ng-container *ngIf="record">
   <header class="row">
-    <h1>{{ record.metadata.title }}</h1>
+
+    <!-- TITLE -->
+    <ng-container *ngIf="record.metadata.title">
+      <ng-container *ngFor="let title of record.metadata.title">
```

Comment:
I would use a property of a method instead of ngFor. This will allows to detect if more than one main title exists.

Reply:
done